client/lib/cgutil/cpuset_manager_v1.go

```diff
@@ -99,15 +108,21 @@ func (c *cpusetManager) CgroupPathFor(allocID, task string) CgroupPathGetter {
 			return "", fmt.Errorf("task %q not found", task)
 		}
 
+		timer, stop := helper.NewSafeTimer(0)
+		defer stop()
+
 		for {
+
 			if taskInfo.Error != nil {
 				break
 			}
+
+			timer.Reset(100 * time.Millisecond)
```

Comment:
We could move this into the `timer.C` case below, I think. Or do we pretty much always need the initial 100ms?